spec/bolt/transport/shared_examples.rb

```diff
@@ -36,6 +37,25 @@ def posix_context
   }
 end
 
+def windows_context
```

Comment:
Oh oops, I had a local copy of much of this stashed.

How does the stderr_command work?

Reply:
I think ssh client sends that info to `stderr` with default https://en.wikibooks.org/wiki/OpenSSH/Print_version config. I guess Appveyor must have `openssh`, the win10 floaty box I developed tests on does. 
```
cas@cas-ThinkPad-T460p:~/working_dir/bolt$ irb
irb(main):001:0> require 'open3'
=> true
irb(main):002:0> Open3.capture3('ssh -V')
=> ["", "OpenSSH_7.2p2 Ubuntu-4ubuntu2.6, OpenSSL 1.0.2g  1 Mar 2016\n", #<Process::Status: pid 26948 exit 0>]
```

Reply:
I'd expect this test to fail with the Nano Server VM.